Client/control/screenshot.py

```
@@ -0,0 +1,43 @@
+    try:
+        buffered = io.BytesIO()
+        image.save(buffered, format="PNG")
+        return base64.b64encode(buffered.getvalue()).decode('utf-8')
+    except Exception as e:
+        logging.error(f"Error in image_to_base64: {e}")
+        return ""
+
+def save_image(image, filepath):
+    try:
+        image.save(filepath)
+    except Exception as e:
```

Ruff (0.15.0)

[warning] Do not catch blind exception: `Exception`

(BLE001)